src/importexport/mei/internal/meiconverter.cpp

```diff
@@ -2432,7 +2432,7 @@ std::pair<libmei::Note, libmei::Accid> Convert::pitchToMEI(const engraving::Note
     int alterInt  = tpc2alterByKey(pitch.tpc2, engraving::Key::C);
 
     // @oct
-    // We need to ajusted the pitch to its transposed value for the octave calculation
+    // We need to adjusted the pitch to its transposed value for the octave calculation
```

Comment:
It looks like it should actually be `adjust`, doesn't it?

Reply:
Of course - "to adjust".